ihatemoney/tests/tests.py

```diff
@@ -63,6 +65,7 @@ def post_project(self, name):
                 "contact_email": f"{name}@notmyidea.org",
                 "default_currency": "USD",
             },
+            follow_redirects=follow_redirects,
```

Comment:
Good question. Before my changes, no call to `post_project` was using the result. So it should make no difference whether redirections are followed or not.

After looking around a bit, a use-case for **not** following redirects is to test where the page redirects to (see https://stackoverflow.com/a/25354698). We currently do that by checking the content of the final page, which is indeed brittle. But fixing that is clearly outside the scope of this PR.

Given this, I can make `follow_redirects=True` the default for `post_project` but I think it makes sense to keep the argument.

Reply:
Actually, the "clean" method to check for redirects doesn't work in our cases (at least the one I checked), because we also need to test that a success/error message is correctly displayed, and that involves following the redirection...

Anyway, I have switched `follow_redirects=True` by default, it should make things simpler.